ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/orchestration.md

```diff
@@ -0,0 +1,186 @@
+# DPP Orchestration Layer with OCI Event and Function: setup guide
+
+## Scenario
+This doc is to show how to setup OCI Events and Functions to make them work in synergy. In this example, the scenario is once a config file in `json` is uploaded to assigned **object storage bucket**, an **event** listening to the bucket will be triggered and it sends a signal to **OCI Function**, which will read the uploaded json and trigger a **OCI Data Flow** job. We will use the [Remove unnecessary column](./remove_unnecessary_columns.md) as the data flow use case here.
+
+## Main OCI Services Involved
+- Object Storage Bucket
+- Oracle Functions
+- Events Service
+- Data Flow
+- Cloud Shell
+
+## Setup steps
+### Create a dynamic group and related policy
```

Comment:
Could you add more details here eg. Go to Governance and Administration>Dynamic Groups